benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+

```

Comment:
Should we set `cmake_minimum_required(VERSION 3.23)` here? (I am confused as to whether every `CMakeLists.txt` file should say that it expects the latest features and settings, or only the top-level one.)